src/geocoding/tests/run_tests.sh

```diff
@@ -5,8 +5,10 @@ docker build -t quantumleap ../../../
 docker-compose up -d
 sleep 8
 
-docker run -ti --rm --network tests_geocodingtests quantumleap pytest geocoding/
+cd ../../../
+pytest src/geocoding/ --cov-report= --cov-config=.coveragerc --cov-append --cov=src/
```

Comment:
Any reason for running pytest outside the container? Is this way easier to integrate with coverage tool maybe?

Note for this pytest command to work in the local environment, we need at least to install the plugins used by this command, and make sure we're using the same pytest version. Could you put those in the pipenv file maybe (instead of installing them in the travis script)? This way we make sure we always install the same set of tools, everywhere. For the time being, we don't mind having the same pipenv file for all environments (testing, dev or prod).

Maybe there's something else that needs to be done locally for the coverage to work (or to be disabled if so wanted)?

And maybe at this point the run_tests scripts are no longer worth much, but that we can discuss other day in another PR :)

Reply:
When the test cases are running by pytest, we have generate the coverage report by pytest-cov. 
Quantumleap code and test cases are in same directory for each submodules (translator, reporter, geocoding etc.).
In previous method of test case execution, following are the command which execute from run_tests.sh script for each submodule:
docker-compose up -d
docker run -ti --rm --network tests_translatorstests quantumleap pytest translators/tests
docker-compose down -v

Through above commands, it will create report for one submodule and if execution of next test case for another submodule, will proceed then the previous report will not be available at that time as docker down command will execute with end of each test case and due to this final report will not be cover the complete result.
Now if we keep the report creation outside of container then we will be able to get the expected report. And we have followed the followed the same docker-compose.yml files for each submodules but test cases are running outside the container.

Yes, i will update the Pipfile for all the dependencies.

Reply:
OK! can you make sure the tests output is still visible after a pytest run both locally and on travis by default?